tests/test_flobject.py

```diff
@@ -655,11 +655,66 @@ class a_2(Boolean):
     )  # noqa: W293
 
 
-def test_accessor_methods_on_settings_object(sample_solver_session):
-    existing = sample_solver_session.file.read.file_type.get_attr("allowed-values")
-    modified = sample_solver_session.file.read.file_type.allowed_values()
+def test_accessor_methods_on_settings_object(load_static_mixer_case):
+    solver = load_static_mixer_case
+
+    existing = solver.file.read.file_type.get_attr("allowed-values")
+    modified = solver.file.read.file_type.allowed_values()
     assert existing == modified
 
-    existing = sample_solver_session.file.read.file_type.get_attr("read-only?")
-    modified = sample_solver_session.file.read.file_type.is_read_only()
+    existing = solver.file.read.file_type.get_attr("read-only?")
+    modified = solver.file.read.file_type.is_read_only()
     assert existing == modified
+
+    assert (
```

Comment:
Just to emphasize that some of these tests fail before the changes,

Reply:
yes, Thank you